deno.json

```diff
@@ -10,7 +10,8 @@
     "@std/bytes": "jsr:@std/bytes@^1.0.2",
     "@std/encoding": "jsr:@std/encoding@^1.0.5",
     "@std/fs": "jsr:@std/fs@^1.0.4",
-    "@std/path": "jsr:@std/path@^1.0.6"
+    "@std/path": "jsr:@std/path@^1.0.6",
+    "$testdata/": "./testdata/"
```

Comment:
This is only used in 2 test cases to demonstrate other plugins can work together with the deno resolver plugin.